Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -145,6 +159,22 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   [self sendEventWithName:@"remoteNotificationsRegistered" body:notification.userInfo];
 }
 
+- (void)handleRegisterUserNotificationSettings:(NSNotification *)notification
+{
+  if (self.requestPermissionsResolveBlock == nil) {
+    return;
+  }
+  UIUserNotificationSettings *notificationSettings = notification.userInfo;
+  NSDictionary *notificationTypes = @{
+    @"alert": @((BOOL)notificationSettings.types & UIUserNotificationTypeAlert),
```

Comment:
Gotcha, alright I played with it on my device and the only change I made was wrapping the bitwise-and in parenthesis before comparing it. Otherwise it works fine 👍